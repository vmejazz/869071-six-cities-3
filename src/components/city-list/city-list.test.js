import React from "react";
import renderer from "react-test-renderer";
import CityList from "./city-list";

const cityes = {
  Amsterdam: [52.38333, 4.9],
  Paris: [48, 53]
};

it(`<CityList /> should render cityes buttons`, () => {
  const tree = renderer
      .create(
          <CityList
            cityes={cityes}
            activeCity={`Paris`}
          />
      )
      .toJSON();

  expect(tree).toMatchSnapshot();
});
