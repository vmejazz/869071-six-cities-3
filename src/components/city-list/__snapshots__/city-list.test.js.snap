// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CityList /> should render cityes buttons 1`] = `<h1 />`;
